modular/gater/errors.go

```diff
@@ -52,6 +52,7 @@ var (
 	ErrInvalidRedundancyIndex = gfsperrors.Register(module.GateModularName, http.StatusInternalServerError, 50035, "invalid redundancy index")
 	ErrBucketUnavailable      = gfsperrors.Register(module.GateModularName, http.StatusForbidden, 50036, "bucket is not in service status")
 	ErrReplyDownloadData      = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50037, "reply the downloaded data to client failed")
+	ErrTaskMsgExpired         = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50038, "the update time of the task has exceed the expire time")
```

Comment:
please update error code doc as well:

https://docs.bnbchain.org/greenfield-docs/docs/api/storage-provider-rest/sp_response#list-of-error-codes